packages/core/src/lib/NftCard/types.ts

```diff
@@ -38,4 +38,14 @@ export interface INftCardProps {
      * custom details component
      */
     customDetails: ReactNode;
+
+    /**
+     * set card background color
+     */
+    backgroundColor: string;
```

Comment:
should this be optional?

Reply:
yeah, nice catch 🙌🏻 

Will make these new props optional ✅